### ISSUE TITLE
多系列area-stack图在yAxis设置max时超出最大值的部分无法显示

### ISSUE THREAD

**Issue by xintq** [apache/echarts#14183]
### Version
5.0.0

### Reproduction link
[https://echarts.apache.org/examples/zh/editor.html?c=area-stack](https://echarts.apache.org/examples/zh/editor.html?c=area-stack)

### Steps to reproduce
将下面的代码复制到重现链接即可重现
```javascript
option = {
    title: {
        text: '堆叠区域图'
    },
    tooltip: {
        trigger: 'axis',
        axisPointer: {
            type: 'cross',
            label: {
                backgroundColor: '#6a7985'
            }
        }
    },
    legend: {
        data: ['邮件营销', '联盟广告', '视频广告', '直接访问', '搜索引擎']
    },
    toolbox: {
        feature: {
            saveAsImage: {}
        }
    },
    grid: {
        left: '3%',
        right: '4%',
        bottom: '3%',
        containLabel: true
    },
    xAxis: [
        {
            type: 'category',
            boundaryGap: false,
            data: ['周一', '周二', '周三', '周四', '周五', '周六', '周日']
        }
    ],
    yAxis: [
        {
            type: 'value',
            max: 2000
        }
    ],
    series: [
        {
            name: '邮件营销',
            type: 'line',
            stack: '总量',
            areaStyle: {},
            emphasis: {
                focus: 'series'
            },
            data: [120, 132, 101, 134, 90, 230, 210]
        },
        {
            name: '联盟广告',
            type: 'line',
            stack: '总量',
            areaStyle: {},
            emphasis: {
                focus: 'series'
            },
            data: [220, 182, 191, 234, 290, 330, 310]
        },
        {
            name: '视频广告',
            type: 'line',
            stack: '总量',
            areaStyle: {},
            emphasis: {
                focus: 'series'
            },
            data: [150, 232, 201, 154, 190, 330, 410]
        },
        {
            name: '直接访问',
            type: 'line',
            stack: '总量',
            areaStyle: {},
            emphasis: {
                focus: 'series'
            },
            data: [320, 332, 301, 334, 390, 330, 320]
        },
        {
            name: '搜索引擎',
            type: 'line',
            stack: '总量',
            label: {
                show: true,
                position: 'top'
            },
            areaStyle: {},
            emphasis: {
                focus: 'series'
            },
            data: [820, 932, 901, 934, 1290, 1330, 1320]
        }
    ]
};
```

### What is expected?
超出 `yAxis` 最大值的部分被截断，剩余的部分也应该正常显示（4.9没有问题）

### What is actually happening?
超出 `yAxis` 最大值的部分全部被截断，图形显示出现空白

![screenshot_2021-02-02_10-13-04](https://user-images.githubusercontent.com/8326471/106542724-4e8adb00-653f-11eb-934c-74ddd8b7e3b9.png)

**Comment by plainheart** [comment]
似乎是个 bug，补充一点问题：
如果设置了 `clip: false` 5.0 中 `areaStyle` 将不显示，只显示 `symbol`

![image](https://user-images.githubusercontent.com/26999792/106542645-28fdd180-653f-11eb-8dcd-3bb17e65230f.png)

这与 4.9 不同

![image](https://user-images.githubusercontent.com/26999792/106542732-50ed3500-653f-11eb-9e76-6888fb67b652.png)

**Comment by xintq** [comment]
Update - Jun 9, 2021
---------------------
I just found that if the following toolbox configuration is removed from the option, then the chart will show normally as expected.  Does it mean the toolbox is conflict with the area clipping? 

```
...
toolbox: {
        feature: {
            saveAsImage: {}
        }
    },
...
```

**Comment by plainheart** [comment]
Have you tried the latest version 5.1.2? If no, please try it and check the issue still exists in this version.

**Comment by xintq** [comment]
> Have you tried the latest version 5.1.2? If no, please try it and check the issue still exists in this version.
Yes, I tried with the latest 5.1.2, and it still exists, and I've submitted a new issue about this in #15106

**Comment by plainheart** [comment]
Closed this since a new issue #15106 was opened.
